forum/backends/mongodb/api.py

```diff
@@ -567,9 +567,14 @@ def handle_threads_query(
             dict[str, Any]: A dictionary containing the paginated thread results and associated metadata.
         """
         # Convert thread_ids to ObjectId
-        comment_thread_obj_ids: list[ObjectId] = [
-            ObjectId(tid) for tid in comment_thread_ids
-        ]
+        comment_thread_obj_ids: list[ObjectId] = []
+
+        for tid in comment_thread_ids:
+            try:
+                thread_id = ObjectId(tid)
+                comment_thread_obj_ids.append(thread_id)
+            except bson_errors.InvalidId:
+                continue
```

Comment:
will this error occur in case of mongo?
and if somehow this mongo func is running for mysql(for which we have added this exception) then we should raise an exception i think because this shouldn't be running for mysql.
correct me if i am wrong.

Reply:
This is because the elastic search retrieves documents from both MySQL and MongoDB, as we are not deleting the indexes on the migration command. The code snipped will filter the documents that are only related to Mongo.

Reply:
Got it!